schema/schema.hh

```diff
@@ -538,6 +537,12 @@ struct schema_static_props {
     }
 };
 
+struct schema_describe_helper {
+    virtual bool is_global_index(const table_id& base_id, const schema& view_s) = 0;
+    virtual bool is_index(const table_id& base_id, const schema& view_s) = 0;
+    virtual schema_ptr find_schema(const table_id& id) = 0;
+};
```

Comment:
Need virtual destructor. And worthwhile to make it a class, not struct, we use structs for pure data aggregates.

Reply:
Done in v2.